src/download-artifact.ts

```diff
@@ -31,6 +37,8 @@ async function run(): Promise<void> {
         `Artifact ${downloadResponse.artifactName} was downloaded to ${downloadResponse.downloadPath}`
       )
     }
+    // output the directory that the artifact(s) was/were downloaded to
+    core.setOutput(Outputs.DownloadPath, resolve(path))
```

Comment:
What is `path` set to if it's not specified in the input?

Reply:
If it's not specified then `path` an empty string. If you call `resolve('')` or `resolve('.')` it outputs the current working directory